examples/onnx2xla.py

```diff
@@ -77,8 +77,9 @@ def onnx_add(a, b, axis=None, broadcast=True):
   if broadcast:
     axis = (a.dim - b.ndim) if axis is None else axis % a.ndim
     assert a.shape[axis:][:b.ndim] == b.shape
-    b_shape = np.ones(a.ndim, dtype='int64').copy()
-    b_shape[axis:axis + b.ndim] = b.shape
+    b_shape = np.ones(a.ndim, dtype='int64')
```

Comment:
I'd actually take a different approach here. Since what you are computing is a shape, I think a better fix would be to `import numpy as onp` and use `onp` to compute the shape.

(Aside: we're moving to the convention `import numpy as np; import jax.numpy as jnp`, so you might want to do that instead.)

Reply:
Yeah that makes sense, updated.